nextjs-13/src/components/Dropdown/index.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { signOut } from 'next-auth/react';
+
+import Dropdown from './';
+
+jest.mock('next-auth/react');
+
+describe('Dropdown', () => {
+  describe('given the menu is displayed', () => {
+    it('shows sign out button', async () => {
+      render(<Dropdown />);
+
+      fireEvent.click(screen.getByTestId('dropdown-toggle'));
+
+      await waitFor(() => {
+        expect(screen.getByTestId('dropdown-menu')).toBeTruthy();
+      });
+
+      expect(screen.getByText('Sign out')).toBeInTheDocument();
+    });
```

Comment:
Let's add a line break for readability (you seemed to hate line breaks 🙈 )

Reply:
Oops 🙈 Done! https://github.com/nimblehq/js-framework-benchmark/pull/104/commits/f7da8bf4902c6ba26297d0388ac4912b860cf013. Eslint not auto add line break when format files so I often forget too haha.